cmd/ndt7-client/internal/emitter/quiet.go

```diff
@@ -0,0 +1,56 @@
+package emitter
+
+import (
+	"github.com/m-lab/ndt7-client-go/cmd/ndt7-client/internal"
+	"github.com/m-lab/ndt7-client-go/spec"
+)
+
+// Quiet acts as a filter allowing summary and error messages only, and
+// doesn't perform any formatting.
+// The message is actually emitted by the embedded Emitter.
```

Comment:
Do I understand correctly that `-quiet -format=json` or `-quiet -format=human`?

Reply:
Yes. In both cases, `-quiet` will only print the summary (+any error) in either JSON or human-readable format.

Reply:
Yes. In both cases, -quiet will only print the summary (+ any errors) in the specified format.

Reply:
Yeah, so it is nice that it is possible to use the client in this way.